docs/client/javascript-mono/StableIdConsistency.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Working With Stable ID
+sidebar_label: Stable ID
+slug: /client/javascript-sdk/stable-id
+---
+
+import GitHubEmbed from "@site/src/components/GitHubEmbed";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import {
+  SDKDocsBuilder,
+  HOOK__SDKDocUpdate,
+} from "../../sdks/_SDKDocsBuilder.mdx";
+
+
+
+## Overview
+
+StableID is a concept in Statsig that allows us to have a consistent identifier for a single device. 
+This allows us to run [experiments on logged out](/guides/first-device-level-experiment) users (Users without a User ID) 
+as well as target gates on the device level rather than the user level.
+
+
+## How it works
+
+The first time you initialize the SDK, we will check if you have a StableID set in local storage. If not, we will generate a new StableID and set it in local storage. 
+Then the next time you initialize the SDK, we will check for the StableID in local storage and reuse it.
```

Comment:
We need to note somehwere:

local storage is only accessible per subdomain, and not accessible on the server side.  If you need an identifier across multiple subdomains or on your backend for identifying anonymous users, you need to manage a cookie yourself.  The sdk does not use any cookies

Reply:
updated